lib/compiler/src/beam_types.hrl

```diff
@@ -51,14 +51,16 @@
 -define(ATOM_SET_SIZE, 5).
 
 -record(t_atom, {elements=any :: 'any' | [atom()]}).
--record(t_float, {elements=any :: 'any' | {float(),float()}}).
--record(t_fun, {arity=any :: arity() | 'any'}).
--record(t_integer, {elements=any :: 'any' | {integer(),integer()}}).
 -record(t_bitstring, {size_unit=1 :: pos_integer()}).
 -record(t_bs_context, {tail_unit=1 :: pos_integer(),
                        slots=0 :: non_neg_integer(),
                        valid=0 :: non_neg_integer()}).
 -record(t_bs_matchable, {tail_unit=1}).
+-record(t_cons, {head=any :: type(), proper=false :: boolean()}).
+-record(t_float, {elements=any :: 'any' | {float(),float()}}).
+-record(t_fun, {arity=any :: arity() | 'any'}).
+-record(t_integer, {elements=any :: 'any' | {integer(),integer()}}).
+-record(t_list, {type=any :: type(), proper=false :: boolean()}).
```

Comment:
You have forgotten to update the comment at the top of the file. It stil says that the atoms `cons` and `list` represent the cons and list types, respectively.

Reply:
Fixed locally, thanks!